infra/cifuzz/clusterfuzz_deployment.py

```diff
@@ -89,8 +89,7 @@ def make_empty_corpus_dir(self, target_name):
 class ClusterFuzzLite(BaseClusterFuzzDeployment):
   """Class representing a deployment of ClusterFuzzLite."""
 
-  BASE_BUILD_NAME = 'build-'
-  COVERAGE_NAME = 'coverage'
+  COVERAGE_NAME = 'latest'
```

Comment:
See my comment below about why I don't think "latest" should be part of any name.